pkg/kubelet/kubelet.go

```diff
@@ -1352,6 +1353,8 @@ func (kl *Kubelet) initializeRuntimeDependentModules() {
 	if kl.enablePluginsWatcher {
 		// Adding Registration Callback function for CSI Driver
 		kl.pluginWatcher.AddHandler("CSIPlugin", csi.RegistrationCallback)
```

Comment:
probably make sense to replace the "CSIPlugin" with pluginwatcherapi.CSIPlugin in this change, how do you think